### DISCUSSION TITLE
Data Set Filters -- "In" is only working for the first entry

### DISCUSSION THREAD

**Discussion by vvaysman** [eclipse-birt/birt#1398] [Q&A]

**Comment by speckyspooky** [comment]
I have created a simple test report based on the "Classic Models".
I used the employees table and filtered with "IN" of 3 values but I get the correct results and cannot confirm your result.
Attached you find my demo and the screen of the result. I have tested it with BIRT 4.12 and BIRT 4.14 (latest built).

Prehaps you can test my report and for further investigaten we need more details which version you have used
and in best case also a simple demo report from your side because there are very different options to use filter "IN" of numbers (int, dec, string, ...).

[data_set_filter.zip](https://github.com/eclipse-birt/birt/files/12448476/data_set_filter.zip)

![2023-08-27 19_12_59-Greenshot Editor](https://github.com/eclipse-birt/birt/assets/41593722/67220f2f-5495-4462-83bf-b7b994bef7a1)
